packages/amplify-category-auth/src/provider-utils/awscloudformation/auth-stack-builder/auth-stack-transform.ts

```diff
@@ -123,10 +124,11 @@ export class AmplifyAuthTransform extends AmplifyCategoryTransform {
           external: true,
         },
       });
+      const projectInfo = getProjectInfo();
```

Comment:
Does getProjectInfo return a clone / readonly version of the object?

Reply:
it returns reference to fresh object with two strings, see https://github.com/aws-amplify/amplify-cli/blob/73eb14a42a1dda2608ba65f84ea629fa417ec8e6/packages/amplify-cli-extensibility-helper/src/helpers/project-info.ts#L11-L16 .

I think I can make `AmplifyProjectInfo` readonly to make it better.